Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -148,9 +146,7 @@ public struct LSTMCell<Scalar: TensorFlowFloatingPoint>: RNNCell, VectorProtocol
     /// - Parameters:
     ///   - inputSize: The number of features in 2-D input tensors.
     ///   - hiddenSize: The number of features in 2-D hidden states.
-    public init(inputSize: Int, hiddenSize: Int,
-                seed: (Int32, Int32) = (Int32.random(in: Int32.min..<Int32.max),
-                                        Int32.random(in: Int32.min..<Int32.max))) {
+    public init(inputSize: Int, hiddenSize: Int, seed: (Int32, Int32) = randomSeedForTensorFlow()) {
```

Comment:
100 columns

Reply:
Sorry about this. I'm addressing it in #319 .